tests/test_tornado.py

```diff
@@ -0,0 +1,324 @@
+import asyncio
+import os
+import socket
+import time
+import typing
+
+from tornado import testing, web
+
+import sprockets_statsd.tornado
+from tests import helpers
+
+ParsedMetric = typing.Tuple[str, float, str]
```

Comment:
As of python 3.9 this is deprecated correct? vs. using builtin tuple. Though I see 3.7 support 🤔

Reply:
Not sure if they are deprecated or not ... I think that I can safely use `from __future__ import annotations` 💭

Reply:
Indeed ... they are deprecated and slated for removal in October of 2025.  From [PEP-585](https://www.python.org/dev/peps/pep-0585/):

> Importing those from typing is deprecated. Due to PEP 563 and the intention to minimize the runtime impact of typing, this deprecation will not generate DeprecationWarnings. Instead, type checkers may warn about such deprecated usage when the target version of the checked program is signalled to be Python 3.9 or newer. It's recommended to allow for those warnings to be silenced on a project-wide basis.
>
> The deprecated functionality will be removed from the typing module in the first Python version released 5 years after the release of Python 3.9.0.